app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,34 @@
+class FriendshipsController < ApplicationController
+
+  def new
+    @friendships = Friendship.new
+  end
+
+  def create
+    user = User.find(params[:id])
+    @friendship = Friendship.create(user_id: current_user.id, friend_id: user.id, confirmed: false)
+    if @friendship.save
+      flash[:notice] = "Friend request sent successfully!"
+      redirect_to users_path
+    else
+      flash[:alert] = "Cannot send friend request"
+      redirect_to users_path
+    end
+  end
+
+  def accept
+    user = User.find(params[:id])
+    current_user.confirm_friend(user)
+    redirect_to users_path, notice: "You are now friend with #{user.name}"
+  end
```

Comment:
Custom methods like this one should be on the model. I suggest you move it to the  Friendship model.
That way, you call the method on `self` and only change the `confirmed` status of the friendship.
Please add the failure case (if there's an error and the friendship can't be accepted)